### ISSUE TITLE
Matrix norm functionality

### ISSUE THREAD

**Issue by skgadi** [josdejong/mathjs#1857]
While working with the matrix norm, I notice it is not working in the following cases. Please look-into it and guide me.

```javascript
math.norm([[1,2],[3,4]],2);
math.norm([[1,2],[3,4]],'-inf');
```

Thank you

**Comment by josdejong** [comment]
Thanks Suresh! Looking at the code, there is no implementation for norm `2` and `-inf` for 2d matrices:

https://github.com/josdejong/mathjs/blob/develop/src/function/arithmetic/norm.js#L156-L204

Anyone interested in implementing these?

**Comment by gwhitney** [comment]
Indeed, `norm(matrix, 2)` has been implemented, and I am not aware of a standard meaning of `norm(matrix, -Infinity), so closing this issue.
